doc/how_to_guides/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -397,13 +397,199 @@ constraints of an individual ``moveit_msgs::msg::MotionPlanRequest`` are
 already satisfied but the ``MoveGroupSequenceAction`` capability doesn't
 implement such a check to allow moving on a circular or comparable path.
 
-See the ``pilz_robot_programming`` package for a `ROS 1 Python script
+To use the MoveGroupSequenceAction and the ``MoveGroupSequenceService`` refer to the :codedir:`the Pilz Motion Planner sequence example <how_to_guides/pilz_industrial_motion_planner/src/pilz_sequence.cpp>`.
+To run this, execute the following commands in separate Terminals:
+
+::
+
+    ros2 launch moveit2_tutorials pilz_moveit.launch.py
+    ros2 run moveit2_tutorials pilz_sequence
+
+For this action and service, the move_group launch file needs to be modify to include these Pilz Motion Planner capabilities.
+The new 
+:codedir:`the pilz_moveit.launch.py <how_to_guides/pilz_industrial_motion_planner/launch/pilz_moveit.launch.py>` 
+is used instead:
+
+::
+
+    moveit_config = (
+       MoveItConfigsBuilder("moveit_resources_panda")
+       .robot_description(file_path="config/panda.urdf.xacro")
+       .trajectory_execution(file_path="config/gripper_moveit_controllers.yaml")
+       .planning_scene_monitor(
+          publish_robot_description=True, publish_robot_description_semantic=True
+       )
+       .planning_pipelines(
+          pipelines=["pilz_industrial_motion_planner"]
+       )
+       .to_moveit_configs()
+    )
+ 
+    # Starts Pilz Industrial Motion Planner MoveGroupSequenceAction and MoveGroupSequenceService servers
+    move_group_capabilities = {
+       "capabilities": "pilz_industrial_motion_planner/MoveGroupSequenceAction pilz_industrial_motion_planner/MoveGroupSequenceService"
+    }
+
+The sequence script creates 2 targets poses that will be reach sequentially.
```

Comment:
```suggestion
The `pilz_sequence.cpp file` creates 2 target poses that will be reached sequentially.
```

... and see if you can directly link to the source code with the `:codedir:` directive.